cmd/ocsp-updater/main.go

```diff
@@ -354,7 +367,12 @@ func (updater *OCSPUpdater) findRevokedCertificatesToUpdate(batchSize int) ([]co
 	var statuses []core.CertificateStatus
 	_, err := updater.dbMap.Select(
 		&statuses,
-		`SELECT * FROM certificateStatus
+		`SELECT
```

Comment:
I also think we should avoid `SELECT *` as a general rule. But I also think that we should probably have one place that's the "canonical" list of fields we currently select, or we risk having components drift out of sync, which could cause bugs if a field is zeroed in one component but not another. For instance, the list of fields to select might live nicely in `sa/model.go`.